src/scss/layout/_error-pages.scss

```diff
@@ -0,0 +1,91 @@
+.error-page {
+  // Background on the error pages needs to be gray because the illustrations disappear to the background if its white.
+  background-color: $color-silver-medium-light;
+
+  // For the error pages the gap on top grows too large if this is left in place.
+  .no-sidebar .main-content {
+    margin-top: 0;
+  }
+
+  .error-page-content__container {
+    @include breakpoint($breakpoint-m) {
+      display: flex;
+    }
+  }
+
+  .error-page__text-container {
+    @include breakpoint($breakpoint-m) {
+      width: 50%;
+    }
+  }
+
+  .error-page__title {
+    margin-bottom: 0;
+  }
+
+  .error-page__description {
+    // TODO: change to use correct font-definition once the PR is merged. Change to body.
```

Comment:
Well done on this thinking ahead! Good job!

Reply:
Thanks :)